praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/domain/NWK.java

```diff
@@ -0,0 +1,50 @@
+package de.muenchen.oss.praktikumsplaner.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class NWK extends BaseEntity {
+
+    @NotNull(message = "Der Vorname ist erforderlich")
+    // TO-DO: Maximales Char Limit vorübergehend auf 255 limitiert
+    @Size(min = 2, max = 255, message = "Der Vorname darf nur {max} Zeichen lang sein")
+    @Column(name = "vorname")
```

Comment:
wie heute besprochen bitte `column` nur dann setzen wenn es unbedingt notwendig ist. In der Regel sollte Spalte und Property zu einander passen.